Modules/socketmodule.c

```diff
@@ -1635,7 +1635,7 @@ getsockaddrarg(PySocketSockObject *s, PyObject *args,
                 Py_TYPE(args)->tp_name);
             return 0;
         }
-        if (!PyArg_ParseTuple(args, "O&i|II",
+        if (!PyArg_ParseTuple(args, "O&i|II:getsockaddrarg",
```

Comment:
indeed it is related to bpo-28261, but see my comment about gettmarg(). ISTM it is also related to this issue.